change/@microsoft-teams-js-c5770fed-2b55-42a7-987b-1618dd76629b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "update changefile and add beta tags to new interfaces",
```

Comment:
Can you please actually change this to:

Added `@beta` tags to new content action-related interfaces.

I don't think we need to mention the change file part for our change log. :)

Reply:
Of course